extensions/amp-loader/0.1/amp-loader.js

```diff
@@ -169,39 +176,37 @@ class LoaderBuilder {
   addLogo_() {
     const {color, content = this.getDefaultLogo_()} = this.getCustomLogo_();
 
-    this.domRoot_
+    this.loaderRoot_
       .querySelector('.i-amphtml-new-loader-logo')
       .appendChild(content);
 
     if (color) {
-      setStyle(this.domRoot_, 'color', color);
+      setStyle(this.loaderRoot_, 'color', color);
     }
   }
 
   /**
-   * Add a gray default placeholder if there isn't a placeholder already and
+   * Add a gray loading background if there isn't a placeholder already and
    * other special cases.
    * @private
    */
-  maybeAddDefaultPlaceholder_() {
+  maybeAddLoadingBackground_() {
     const hasPlaceholder = !!this.element_.getPlaceholder();
     const hasPoster = this.element_.hasAttribute('poster');
     if (hasPlaceholder || hasPoster) {
       return;
     }
 
     // Is it whitelisted for default placeholder?
-    const tagName = this.element_.tagName.toUpperCase();
+    const {tagName} = this.element_;
     if (
-      DEFAULT_PLACEHOLDER_WHITELIST_NONE_VIDEO[tagName] || // static white list
-      isIframeVideoPlayerComponent(tagName) // regex for various video players
+      !DEFAULT_LOADING_BACKGROUND_WHITELIST_NON_VIDEO[tagName] &&
+      !isIframeVideoPlayerComponent(tagName)
```

Comment:
nit: I'd use a helper function to test whether the tag name is whitelisted and remove the comment for better readability, also then it would be possible to combine this if condition with the one above.